index.html

```diff
@@ -184,9 +183,42 @@ <h6> I can help you build a product, feature or website Look through some of my
                                 <ul>
                                 </div>
                               </div>
-                      </div>
-                          
-                  
+                            </div> 
+                            <section>
+                              <p class="about-p">
+                                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                              </p>
+                              <form action="https://formspree.io/f/xwkjrzvw"
+                              method="POST">
+                                <ul class="inputs">
+                                   <li>
+                                   <label for="name"></label>
+                                    <input type="text" id="name" name="user_name" placeholder="Full name"/>
+                                  </li>
+                                  <li>
+                                    <label for="email"></label>
+                                    <input type="email" id="mail" name="email" placeholder="Email address" />
+                                  </li>
+                                  <li>
+                                    <label for="msg" aria-placeholder=""></label>
+                                    <textarea id="msg" name="msg" placeholder="Hey I’d like to connect to..."></textarea>
+                                  </li>
+                                </ul>
+                                <button class="button-text">Get in touch  </button>
```

Comment:
- [ ] kindly add button  interaction of active (when it's pressed `#008552`) like in figma version

![Screenshot (936)](https://user-images.githubusercontent.com/65298048/229868311-3315303c-2c2b-4652-b177-74330746b558.png)